lib/take_away.rb

```diff
@@ -0,0 +1,17 @@
+class TakeAway
+  attr_reader :menu   
+  def initialize menu = []
+    @menu = menu
+  end
+  def add_menu new_menu
+    @menu << new_menu
+  end
+  def place_order order 
+    @order = order.meals
+  end
+
+  def has_order?
```

Comment:
Style/PredicateName: Rename `has_order?` to `order?`. (https://github.com/bbatsov/ruby-style-guide#bool-methods-qmark)